circle.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+workflows:
+  version: 2
+  linux-builds:
+    jobs:
+      - build-x86_64
+      - build-i686
+
+jobs:
+  build-x86_64:
+    docker:
+#      - image: ubuntu:14.04
+      - image: circleci/python:2.7
```

Comment:
Would it make sense to use Elliot's images for this? They have all of the dependencies needed to build Julia and you can do both x86 and x86-64 rather than doing a 32-bit build on a 64-bit image.